pkg/proxy/ipvs/proxier.go

```diff
@@ -1656,6 +1667,18 @@ func openLocalPort(lp *utilproxy.LocalPort) (utilproxy.Closeable, error) {
 			return nil, err
 		}
 		socket = conn
+	case "sctp":
+	  //SCTP is not supported by golang/net, or any other built-in lib,
```

Comment:
mis-indented whitespace; also there should be a single space after the // in comments.